proxies/sandbox/apiproxy/targets/sandbox.xml

```diff
@@ -91,6 +91,10 @@
                     response.status.code = 400 and response.content Like "*Templates required in *" and response.content Like "* routing config not found*"
                 </Condition>
             </Step>
+            <Step>
+                <Name>RaiseFault.500FailedWriteToDynamoDB</Name>
+                <Condition>response.status.code = 500 and response.content Like "*Error writing request items to DynamoDB*"</Condition>
```

Comment:
We may want to add a `Prefer: code=500` for triggering on the sandbox - this can then be added into the docs!